<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTcpDumpClass</class>
 <widget class="QMainWindow" name="QtTcpDumpClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTcpDump</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="m_combox_macs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ip src 192.168.0.123 or ip dst 192.168.0.123</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>filter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtTcpDump.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_start</sender>
   <signal>clicked()</signal>
   <receiver>QtTcpDumpClass</receiver>
   <slot>slot_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_stop</sender>
   <signal>clicked()</signal>
   <receiver>QtTcpDumpClass</receiver>
   <slot>slot_stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_start()</slot>
  <slot>slot_stop()</slot>
 </slots>
</ui>
